scripts/validate.py

```diff
@@ -16,6 +19,62 @@
         validate.py ./cfr.json ./schema/cfr.schema.json
 """
 
+def intersect_op(listA, listB):
+    set(listA).intersection(listB)
+
+def validate_targeting(targeting):
+    jexl = JEXL()
+    context = Context({
+        'firefoxVersion': 77,
+        'totalBookmarksCount': 0,
+        'userPrefs': {},
+        'addonsInfo': {},
+        'topFrecentSites': {},
+        'pageLoad': 0,
+        'recentVisits': {},
+        'activeExperiments': [],
+        'blockedCountByType': {
+            'fingerprinterCount': 100
+        },
+        'usesFirefoxSync': True,
+        'hasAccessedFxAPanel': True,
+        'currentDate': '2020-04-20',
+        'profileAgeCreated': 1000,
+        'messageImpressions': {
+            'WHATS_NEW_BADGE_75': [100]
+        },
+        'browser': {
+            'messaging-system': {
+                'whatsNewPanel': {
+                    'enabled': True
+                }
+            }
+        }
+    })
+    jexl.add_binary_operator('intersect', 40, intersect_op)
+    jexl.add_transform('date', lambda x: 0)
+    jexl.add_transform('length', lambda x: 0)
+    jexl.add_transform('preferenceValue', lambda x: False)
+    jexl.add_transform('mapToProperty', lambda x, y: [])
+    jexl.add_transform('keys', lambda x: [])
+    jexl.add_transform('bucketSample', lambda x, y, z, q: False)
+
+    jexl.evaluate(targeting, context)
+
+def validate_item_targeting(item):
+    print("Validate", item['id'])
+    try:
+        if ('targeting' in item):
+            validate_targeting(item['targeting'])
+        else:
+            validate_targeting(item['filter_expression'])
+    except Exception as e:
+        if ('targeting' in item):
+            print("Error validating", item['targeting'])
+        else:
+            print("Error validating", item['filter_expression'])
+        print("If the expression includes new targeting attributes or "\
+                "operators those need to be added to the evaluation context:", e)
```

Comment:
You might want to propagate the exception here, otherwise, CI won't treat this as an Error.

Calling `sys.exit(1)` should just do the work.